Content.Client/Entry/IgnoredComponents.cs

```diff
@@ -5,6 +5,8 @@ public static class IgnoredComponents
     {
         public static string[] List => new[]
         {
+            "AirlockPainter",
+            "Anchorable",
```

Comment:
This was accidentally introduced when merging master & fixing conflicts. Anchorable should no longer be here.
```suggestion
```